pinot-query-runtime/src/test/resources/queries/CountDistinct.json

```diff
@@ -119,11 +119,13 @@
       {
         "comments": "table aren't actually partitioned by val thus all segments can produce duplicate results, thus [[8]]",
         "sql": "SELECT SEGMENT_PARTITIONED_DISTINCT_COUNT(val) FROM {tbl1}",
+        "ignored": true,
```

Comment:
what's the reason behind this?

Reply:
Not this one, but the other tests ignored fail just because I added more tables. It seem the test framework is not able to specify how to partition data and by adding tests with new tables we modify the default partition distribution.

You can try to remove the `ignored`  tests here and run the tests with and without my new tests and you will see that the results are different

Reply:
ah i see. i recently fixed this in #11937 . PTAL

Reply:
I've merged my branch with master and it didn't fix the issue. In fact now we have a third test here that fails. I'm going to remove the ignores so the issue is more evident.

Reply:
Shall we try removing the ignored flag and see if they still fail? If so, we might want to debug them because this can cause regression

Reply:
i tried it and it didnt' work. let's address separately